style.css

```diff
@@ -0,0 +1,273 @@
+  align-items: center;
+  width: 100%;
+  height: 44px;
+}
+
+.toolbar button {
+  margin-right: 25px;
+  border: none;
+  background: none;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+}
+
+.title h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.title h1 span {
+  color: #36b37e;
+}
+
+.description {
+  padding: 0 20px;
+  margin: 5% 0%;
+}
+
+.description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.socialIcons ul {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+  gap: 20px;
+}
+
+.action {
+  margin-top: 100px;
+}
+
+/* .works_content {
+  margin-top: 25px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.works_content h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 0 25px;
+  margin-bottom: 15px;
+}
+
+.works_cards .card {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  outline: 1px solid #eee;
+  border-radius: 7px;
+}
+
+.card_image {
+  height: 50%;
+}
+
+.card_image img {
+  height: 100%;
+  width: 100%;
+}
+
+.card_content {
+  height: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.card_content h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.card_content ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.card_content ul li {
+  background-color: #eee;
+  padding: 8px 12px;
+  font-size: 12px;
+  color: #3a4a42;
+}
+
+/* About */
+
+/* .about {
+  margin-top: 25px;
+}
+
+.about_content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.about_content h3 {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-right: 25px;
+  margin-bottom: -40px;
+}
+
+.about_content p {
+  color: #42526e;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  margin-bottom: 15px;
+  padding: 0 15px;
+}
+
+.about_boxes {
+  margin-top: 50px;
+}
+
+.box {
+  width: 327px;
+  height: 316px;
+  background: #ebf0ee;
+  margin-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 10px;
+}
+
+.box h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+}
+
+.box ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  gap: 10px;
+}
+
+.box ul li {
+  background: #fff;
+  padding: 8px 12px;
+  font-size: 12px;
+  color: #36b37e;
+  border-radius: 8px;
+} */
+
```

Comment:
[OPTIONAL]
- [ ] Kindly delete this block of commented code, as it affects badly the organization and look of your code.